src/harness/testcases/WINNF_FT_S_FDB_testcase.py

```diff
@@ -585,8 +577,15 @@ def test_WINNF_FT_S_FDB_3(self, config_filename):
                                     grant_request_g,
                                     config['conditionalRegistrationData'])
 
-    # TODO
     # Step 2: Create FSS database which includes atleast one FSS site near location 'X'.
+    fss_database = DatabaseServer(
```

Comment:
Line   number 581 will prevent code from executing because import of Database file   is missing and you are trying to access DatabaseServer      E.g. Need to add the below line where imports are added.     from database import DatabaseServer

Reply:
Rebased to master which includes the import.